isis/tests/cnetbin2pvlTests.cpp

```diff
@@ -0,0 +1,41 @@
+#include <QTemporaryDir>
+#include <iostream>
+
+#include "Pvl.h"
+#include "cnetbin2pvl.h"
+#include "Fixtures.h"
+#include "Progress.h"
+#include "gmock/gmock.h"
+#include "FileName.h"
+#include "UserInterface.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/cnetbin2pvl.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestCnetbin2pvlDefault) {
+  QTemporaryDir prefix;
+  ASSERT_TRUE(prefix.isValid());
+
+  QString pvlOut = prefix.path()+"/cnetbin2pvlNetwork.pvl";
+  QString cnetFilePath = prefix.path()+"/cnetbin2pvl_network.net";
+  FILE* cnetFile;
+  cnetFile = fopen(cnetFilePath.toStdString().c_str(), "wb");
+  fwrite(network, sizeof(char), sizeof(*network), cnetFile);
+  fclose(cnetFile);
```

Comment:
This isn't how binary networks are written out. They have a specific on disk binary format. You can take the PVL network in the data directory and run it through cnetpvl2bin to convert it to binary.

Reply:
I definitely made a mistake here.  The "from" parameter from the UI requires a path to a control network, but the pvl is written based on the "network" parameter that's passed to the function. It doesn't look like "from" is used at all, so there's no need to write anything to disk.